inlong-sort/sort-formats/format-rowdata/format-rowdata-base/src/main/java/org/apache/inlong/sort/formats/base/DefaultDeserializationSchema.java

```diff
@@ -110,7 +125,7 @@ public boolean skipCurrentRecord(T element) {
         return ignoreErrors && errorOccurred;
     }
 
-    protected abstract T deserializeInternal(byte[] bytes) throws IOException;
+    protected abstract T deserializeInternal(byte[] bytes) throws Exception;
```

Comment:
Why expand IOException to Exception？

Reply:
failed handler throw Exception, not IOException